apis/v1beta1/gateway_types.go

```diff
@@ -327,7 +327,7 @@ type GatewayTLSConfig struct {
 	//
 	// Support: Core - A single reference to a Kubernetes Secret of type kubernetes.io/tls
 	//
-	// Support: Implementation-specific (More than one reference or other resource types)
+	// Support: Extended (More than one reference or other resource types)
```

Comment:
I think this is going to need some more discussion. If we say that supporting multiple TLS certificates is "Extended" support, we'll need to define how multiple TLS certs should be handled. I'm not entirely sure what that looks like today, but it could be worth getting some input from anyone currently supporting that.

One concrete change here would be to limit the scope of "Extended" support to Kubernetes TLS secrets. That would result in the following:

```
	// Support: Core - A single reference to a Kubernetes Secret of type kubernetes.io/tls
	//
	// Support: Extended - Multiple references to Kubernetes Secrets of type kubernetes.io/tls
	//
	// Support: Implementation-specific - References or other resource types
```

Reply:
I've added an agenda item for next week's community meeting to discuss this and bump this old PR.

Reply:
I really like @robscott's suggestion